web/stories/Atoms/Tag/TagStatus.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { Tag } from "@/components/njwds-extended/Tag";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+
+export default {
+  title: "Atoms/Tag/Status",
+  component: Tag,
+  decorators: [withDesign],
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/vAa8neaM0JJSmldck5vlBC/BFS-Design-System-(Sprint-33%2B)?node-id=1%3A1469",
```

Comment:
does the url for figma have to point to the tag section? https://www.figma.com/file/vAa8neaM0JJSmldck5vlBC/BFS-Design-System?type=design&node-id=2204-2427&mode=design&t=rXBjJuzmv6z3lNFl-0

Reply:
I mean, I think it should given the component we're working with?

Reply:
You should be able to copy the URL to the specific frame in Figma that houses the tag components and paste it into the section. There is a design tab within storybook that allows you to see what that actual page looks like.

Link - https://www.figma.com/file/vAa8neaM0JJSmldck5vlBC/BFS-Design-System?type=design&node-id=2895%3A6988&mode=design&t=yjn1vHzCSN7UAIMG-1

You can grab the URL by selecting the frame and then hitting the share button
<img width="718" alt="Screenshot 2023-11-15 at 3 41 39 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/6afbcd30-45ad-42b2-a103-b961e0902681">



Once the link has been pasted into the options, it should render something like the following

<img width="875" alt="Screenshot 2023-11-15 at 3 42 26 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/65919b1b-660b-4dbe-be0e-c2b115f916c4">